Cargo.toml

```diff
@@ -25,6 +25,7 @@ harness = false
 num-complex = "0.2.4"
 once_cell = "1.4.0"
 rustfft = "3.0.1"
+libc = "0.2"
```

Comment:
Could this be behind a feature flag?

Reply:
It would require a new cargo-c, but can be done. (I'm using nnnoiseless to push some boundaries).